index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wamaya portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    <header>
+    <nav>
+        <a id="logo" href="#">My logo</a>
+        <ul id="menu">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><a href="#"><img src="media/Icon - Mail.png" alt="Email"></a></li>
+        </ul>
+        <img src="media/Union.png" alt="menu">
+    </nav>
+    </header>
+    <section id="headline">
+        
+        <aside id="social" >
+            <ul >
+                <li><a href="#"><img src="media/github.png" alt="github"></a></li>
+                <li><a href="#"><img src="media/Linkedin icon.png" alt="linkedIn"></a></li>
+                <li><a href="#"><img src="media/angel list.png" alt="angel list"></a></li>
+                <li><a href="#"><img src="media/twitter.png" alt="twitter"></a></li>
+                <li><a href="#"><img src="media/medium.png" alt="medium"></a></li>          
+        </ul>
+        </aside>
+        <div id="bio">
+        <h1>Hey There. <br> I'm Sylvester</h1>
+        <h2>I'm a Software Developer</h2>
+        <p>I can help you build a product , feature or website. Look through some of 
```

Comment:
[OPTIONAL] Good job with using p element for this area. But since the previous reviewer didn't notice, I will keep it as optional, let's align vertically with an equal value on the left and right of it like on the Figma template.
<img width="416" alt="Screenshot 2023-03-24 at 8 10 11 PM" src="https://user-images.githubusercontent.com/25887253/227596306-51797ea8-354c-4f83-acdd-58d5430627ea.png">
<img width="416" alt="Screenshot 2023-03-24 at 8 13 45 PM" src="https://user-images.githubusercontent.com/25887253/227596313-ebc85bf3-0224-473d-9d0e-82d34cc9025c.png">